.gitlab-ci.yml

```diff
@@ -16,4 +16,5 @@ test:
         - echo "Testing"
         - cd ui
         - yarn install
+        - yarn lint
```

Comment:
yarn lint? co to? nawet w dokumentacji yarn nie widzę takiej komendy

Reply:
Look here https://github.com/no-waste-app/no-waste/pull/1/files#diff-193b27d62e4fbda3d563009fed5ec6761a05f73558d94b39fab63ae948c679eaR11
Yarn allows you to call scripts without `run` explicitly. `yarn lint` is an equivalent to `npm run lint`.